src/templates/P_Dashboard._nodeFormOptions.php

```diff
@@ -1,4 +1,5 @@
 <div class="form-group">
+    <label>Visibility</label>
 	<div class="c-radio-btn">
 		<div class="form-check">
```

Comment:
Don't forget to convert your leading tabs to spaces.  It helps to set your IDE to display whitespace.